android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -772,9 +776,9 @@ private void initializePlayerCore(ReactExoplayerView self) {
         player.addListener(self);
         player.setVolume(muted ? 0.f : audioVolume * 1);
         exoPlayerView.setPlayer(player);
-        if (adsLoader != null) {
-            adsLoader.setPlayer(player);
-        }
+
+        adsLoader.setPlayer(player);
+
```

Comment:
We can have this without checks, if we disable ads (in example they are enabled by default!) this will cause error - but my past _me_ left us an great option
```
if (BuildConfig.USE_EXOPLAYER_IMA) {
    adsLoader.setPlayer(player);
}
```
With this we can check if ads are enabled

Reply:
All Ads related "things" we can wrap with this but it's optional I guess - only setPlayer cause playback error

Reply:
Yep I know, but as this is just empty function in the stub file, I think this is not necessary, it will just overload the code... (I made the choice not to add it). Do you really think I should add these additionnel tests ?
The stub will need to be implemented anyway ...

Reply:
OK @KrzysztofMoch I handled your point in: https://github.com/TheWidlarzGroup/react-native-video/pull/4220